resources/blueprints/product.yaml

```diff
@@ -78,6 +78,13 @@ sections:
               handle: variant_attributes
               field:
                 fields:
+                  -
```

Comment:
Good catch! I added the hidden field to my project's blueprint but not the Simple Commerce one.